secretariat/management/commands/import-from-outline.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from config.settings import OUTLINE_API_URL
+from secretariat.utils.outline import Client as OutlineClient
+from secretariat.utils.outline import RemoteServerError
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    help = "Imports users from Outline. Creates missing users in Django, updates existing users if needed."
+
+    def handle(self, *args, **options):
+        client = OutlineClient()
+
+        self.stdout.write(f"Importing users from instance {OUTLINE_API_URL}.")
+
+        try:
+            outline_users = client.list_users()
+        except RemoteServerError:
+            self.stdout.write(
+                self.style.ERROR("Couldn't not reach remote server. Exiting.")
+            )
+
+        known_emails = [user.email for user in User.objects.all()]
```

Comment:
Là c'est pas grave parce qu'on a peu d'utilisateurs déjà existants en base, mais usuellement il vaut mieux éviter d'utiliser `.all()` en production.

Ici si tu veux juste extraire la liste des e-mails connus, plutôt chercher du côté de [values_list](https://docs.djangoproject.com/fr/4.2/ref/models/querysets/#values-list). Ça donnerait peut-être qch du genre : 

```suggestion
        known_emails = [value[0] for value in User.objects.values_list("email")]
```

Reply:
Ah d'acc ! Est-ce que tu sais pourquoi il vaut mieux pas utiliser `.all()`en production ?

Reply:
J'avais récup la liste des emails parce que ça me paraissait justement plus efficace de récupérer une liste de mails et de chercher dedans que d'appeler User.objects.filter(email=user['email'].exists() à chaque itération. 

Mais si ça te semble pas une bonne idée, on peut changer !